README.md

```diff
@@ -112,4 +112,4 @@ We appreciate all contributions to improve MMPose. Please refer to [CONTRIBUTING
 
 MMPose is an open source project that is contributed by researchers and engineers from various colleges and companies.
 We appreciate all the contributors who implement their methods or add new features, as well as users who give valuable feedbacks.
-We wish that the toolbox and benchmark could serve the growing research community by providing a flexible toolkit to reimplement existing methods and develop their own new models.
\ No newline at end of file
+We wish that the toolbox and benchmark could serve the growing research community by providing a flexible toolkit to reimplement existing methods and develop their own new models.
```

Comment:
what's the diff of this line?

Reply:
I can't find and I think it changed by pre-commit